backend/app/adapter/sqldb/changelog_integration_test.go

```diff
@@ -269,6 +269,105 @@ func TestChangeLogSql_DeleteChange(t *testing.T) {
 	}
 }
 
+func TestChangeLogSql_UpdateChange(t *testing.T) {
+	summaryMarkdown1 := "summary 1"
+	summaryMarkdown2 := "summary 2"
+	summaryMarkdown3 := "summary 3"
+
+	testCases := []struct {
+		name              string
+		tableRows         []changeLogTableRow
+		change            entity.Change
+		expectedChangeLog []entity.Change
+	}{
+		{
+			name: "update an existing change",
+			tableRows: []changeLogTableRow{
+				{
+					id:              "12345",
+					title:           "title 1",
+					summaryMarkdown: summaryMarkdown1,
```

Comment:
```suggestion
					summaryMarkdown: "summary 1",
```
When possible, please fill in the input data in the test case themselves directly. This will help readers focus on understanding the test case. The places we used variables were because the field require a pointer and we can only take the pointer of a variable.

Reply:
@byliuyang , we are still using the pointers to these variables below to construct `expectedChangeLog`

Reply:
@rohithbalaji123 I see. Then fine.